README.md

```diff
@@ -439,6 +439,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [GitHub Actions for Rails Developers](https://www.youtube.com/watch?v=gGUXydw22zw)
 - [GitHub Actions Advent Calendar](https://www.edwardthomson.com/blog/github_actions_advent_calendar.html)
 - [Zero Downtime Laravel Deployments with GitHub Actions](https://atymic.dev/blog/github-actions-laravel-ci-cd/)
+- [Video: Five Ways to Automate with GitHub Actions](https://youtu.be/eB0nUzAI7M8)
```

Comment:
Great video! Approved, but you will have to pull master and push again since a broken linter has since been disabled.